### PR TITLE
implement loading image, segmentation and tissue names/colors from CLI

### PR DESCRIPTION
**Pull request by dyollb**
implement loading image, segmentation and tissue names/colors from CLI
- each option is optional and can be used wo the others

Example use:
```
iSeg.exe -I "C:\Users\Beta\folder\T1.nii.gz" -L "C:\Users\Beta\folder\labels.nii.gz" -T "C:\Users\folder\folder\labels_16.txt"
```

or wo the pre-existing segmentation:
```
iSeg.exe -I "C:\Users\Beta\folder\T1.nii.gz" -T "C:\Users\folder\folder\labels_16.txt"
```

### CODE REVIEW

File: iSeg/MainWindow.cpp [hunk 1]
```diff
@@ -5675,7 +5684,7 @@ void MainWindow::Slices3dChanged(bool new_bitstack)
 
 	for (size_t i = 0; i < m_Tabwidgets.size(); i++)
 	{
-		((WidgetInterface*)(m_Tabwidgets[i]))->NewLoaded();
+		//((WidgetInterface*)(m_Tabwidgets[i]))->NewLoaded();
```
Comment (mguidon) [inline iSeg/MainWindow.cpp:5687]: Delete?

File: iSeg/MainWindow.cpp [hunk 2]
```diff
@@ -3776,6 +3771,76 @@ void MainWindow::ExecuteExporttissueindex()
...
+		emit BeginDatachange(data_selection, this, false);
+
+		m_Handler3D->ReloadImage(loadfilename.toAscii(), m_Handler3D->StartSlice());
+		ResetBrightnesscontrast();
+
+		emit EndDatachange(this, iseg::ClearUndo);
+	}
+}
+
 void MainWindow::ExecuteLoadtissues()
```
Comment (mguidon) [inline iSeg/MainWindow.cpp:3844]: Capital T?

**Review by mguidon — APPROVED** [review]
LGTM only minor comments.

File: iSeg/MainWindow.cpp (hunk 2 above)
Comment (dyollb) [reply iSeg/MainWindow.cpp]: well, fixed, but still "wrong" in many similar functions/

File: iSeg/MainWindow.cpp (hunk 1 above)
Comment (dyollb) [reply iSeg/MainWindow.cpp]: thanks. uncommented again

**Pull request merged**